Gds/src/fprime_gds/common/history/chrono.py

```diff
@@ -40,17 +40,17 @@ def __init__(self, filter_pred=None):
 
         self.retrieved_cursor = 0
 
-    def data_callback(self, data_object, sender=None):
```

Comment:
Why?

Reply:
Pylint was complaining that "Parameters differ from overridden 'default' method". (where the variable is data)
This is somewhat of a false positive as the difference between 'data' and 'data_object' is practically zero.

However Ashley in [this issue](https://github.com/PyCQA/pylint/issues/1631) makes a good point on the subject that 

"Why do you consider this a false positive? Renaming arguments means that as a user I could no longer do something like...
```
encoder = MyEncoder()
kwargs = {'o': to_encode}
encoder.default(**kwargs)
```

Reply:
Nope, I am happy with the answer.  I agree with Ashley too.